<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetNfe</class>
 <widget class="QWidget" name="WidgetNfe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidgetNfe">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabEntrada">
      <attribute name="title">
       <string>Entrada</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_44">
       <item row="0" column="0">
        <widget class="WidgetNfeEntrada" name="widgetEntrada" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSaida">
      <attribute name="title">
       <string>SaÃ­da</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_45">
       <item row="0" column="0">
        <widget class="WidgetNfeSaida" name="widgetSaida" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetNfeSaida</class>
   <extends>QWidget</extends>
   <header>src/widgetnfesaida.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetNfeEntrada</class>
   <extends>QWidget</extends>
   <header>src/widgetnfeentrada.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
